js/src/wp-seo-post-scraper.js

```diff
@@ -328,9 +331,11 @@
 				indicator.className = 'na';
 			}
 
-			$( '.yst-traffic-light' )
-				.attr( 'class', 'yst-traffic-light ' + indicator.className )
-				.attr( 'alt', indicator.screenReaderText );
+			$trafficLight.attr({
+				'class': 'yst-traffic-light ' + indicator.className,
```

Comment:
When using the arguments in an object they shoudn't be surrounded by quotes `class` instead of `'class'`